src/components/PlantsV2/components/HectaresPlantedCard.tsx

```diff
@@ -38,10 +38,10 @@ export default function HectaresPlantedCard({ plantingSiteId }: HectaresPlantedC
       isEditable={false}
       contents={
         <Box display='flex' flexDirection='column'>
-          <Typography fontSize='16px' fontWeight={600} marginBottom={theme.spacing(5)}>
+          <Typography fontSize='16px' fontWeight={600} marginBottom={theme.spacing(5)} sx={{ flexFlow: 'row wrap' }}>
```

Comment:
I think you should be able to remove these flex properties from the Typography component since it is not a flex box, vercel shows these properties as invalid. Seems to be working without this property. Or.. use word-break/white-space/width properties to get what you want. Removing for now may be ok?